BluetoothLeChat/app/src/main/java/com/example/bluetoothlechat/bluetooth/Message.kt

```diff
@@ -0,0 +1,6 @@
+package com.example.bluetoothlechat.bluetooth
+
+sealed class Message(val text: String) {
+    class RemoteMessage(text: String) : Message(text)
```

Comment:
sorry if i missed it, but might be helpful to add some comments about what a `remoteMessage` vs `localMessage` is